### PR TITLE
Poolchems

### PR DESCRIPTION
**Pull request by yobob591**
Makes it so if you put a large beaker into the pool controller containing 100 units of one reagent it will put dat shit in the pool. Will likely need balancing in the future, but so far chems absorb into you slow enough that you can generally escape even deadly chems like chloral. I also dont know how to get rid of the SMES log fix shit because github is fucking confusing as shit

### CODE REVIEW

**Comment by yobob591** [comment]
help i fucking hate this github bullshit whats even happening

**Comment by yobob591** [comment]
my github is completely fucked i cant commit shit anymore and it stuck halfway through a merge and is barfing errors at me aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

**Comment by Kokojos** [comment]
@yobob591 I can help but only if you use sourcetree, text me on slack.

**Comment by Gandalf2k15** [comment]
code/modules/power/smes.dm:161:error: missing left-hand argument to <<. Looks like a fairly simple fix.

File: code/modules/power/smes.dm [hunk 1]
```diff
@@ -158,10 +158,19 @@
 		terminal.dismantle(user)
 
 	//crowbarring it !
+<<<<<<< HEAD
```
Comment (Gandalf2k15) [inline code/modules/power/smes.dm:161]: wot

**Comment by yobob591** [comment]
I just need to unfuck github and get rid of the SMES fix as someone else already fixed it

**Comment by Nebyab** [comment]
I mean there are balance concerns, but I think its worth checking out

**Comment by dreamcrab** [comment]
i really enjoyed this back when it sorta worked before, if this runs into the same problems as last time it will be wholly ineffective but fun nonetheless

**Comment by yobob591** [comment]
i will fix the github shit when i have the motivation to slave away for a few more hours because god damn i spent like 8 hours trying to get this pr together and it was fucked by git

**Comment by yobob591** [comment]
and 8 hours isnt a exaggeration, i literally spent around 7-8 hours fucking with github trying to figure this shit out

**Comment by Carbonhell** [comment]
PR inactive since a week,closing,if you're still working on it just write here and i'll reopen it

**Pull request closed without merging**